<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindInPageBar</class>
 <widget class="QFrame" name="FindInPageBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>43</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="findEdit">
     <property name="placeholderText">
      <string>Find in page</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fNextButton">
     <property name="icon">
      <iconset resource="../resources/kiwix.qrc">
       <normaloff>:/icons/search_forward.svg</normaloff>:/icons/search_forward.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fPreviousButton">
     <property name="icon">
      <iconset resource="../resources/kiwix.qrc">
       <normaloff>:/icons/search_backward.svg</normaloff>:/icons/search_backward.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>350</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="hideButton">
     <property name="icon">
      <iconset resource="../resources/kiwix.qrc">
       <normaloff>:/icons/close.svg</normaloff>:/icons/close.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/kiwix.qrc"/>
 </resources>
 <connections/>
</ui>
